recipes/wise_enum/all/conanfile.py

```diff
@@ -0,0 +1,43 @@
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+class WiseEnumConan(ConanFile):
+    name = "wise_enum"
+    description = "Header-only C++11/14/17 library provides static reflection for enums, work with any enum type without any boilerplate code."
+    topics = (
+        "conan",
+        "cplusplus",
+        "enum-to-string",
+        "string-to-enum"
+        "serialization",
+        "reflection",
+        "header-only",
+        "compile-time"
+    )
+    url = "https://github.com/quicknir/wise_enum"
+    homepage = url
+    license = "BPL"
+    settings = "compiler"
+    no_copy_source = True
+    
+    @property
+    def _source_subfolder(self):
+        return "src"
+    
+   
+
+    def configure(self):
+        if self.settings.compiler.get_safe("cppstd"):
+            tools.check_min_cppstd(self, "11")
+    def package_id(self):
+        self.info.header_only()
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = self.name + "-" + self.version
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def package(self):
+        self.copy("*.h", dst="include", src=self._source_subfolder)
+        self.copy("LICENSE", dst="licenses" , src=self._source_subfolder)
```

Comment:
We explicitly set the names for the generator when the upstream project creates one.
https://github.com/quicknir/wise_enum/blob/34ac79f7ea2658a148359ce82508cc9301e31dd3/CMakeLists.txt#L66
```suggestion
        self.copy("LICENSE", dst="licenses" , src=self._source_subfolder)
        
    def package_info(self):
        self.cpp_info.names["cmake_find_package"] = "wise_enum"
        self.cpp_info.names["cmake_find_package_multi"] = "wise_enum"
```